verify.go

```diff
@@ -26,9 +28,30 @@ type Server struct {
 // New creates a new Server.
 func New(options ...Option) *Server {
 	cfg := getConfig(options...)
+
+	var verifierOpts tlsVerifierOptions
+	if len(cfg.extraCACerts) > 0 {
+		pool, err := x509.SystemCertPool()
+		if err != nil {
+			panic(err)
```

Comment:
Should we `log.Fatal()` here, for consistency with the other error handling logic below?